src/components/graphqlFormGenerator/FormGenerator.vue

```diff
@@ -0,0 +1,113 @@
+<template>
+  <v-form validate>
+    <h3>{{ mutation.name }}</h3>
+    <p>{{ mutation.description }}</p>
+    <form-input
+     v-for="input in inputs"
```

Comment:
Testing my understanding - `inputs` is the list of buttons?

Reply:
Yeah, by input I mean a generic user input like a text box, radio buttons or whatnot.

These "inputs" are the arguments to the GraphQL mutation.